CollapseLauncher/Classes/Properties/AppActivation.cs

```diff
@@ -1,8 +1,10 @@
 ﻿using System;
 using System.Linq;
 using System.Text.RegularExpressions;
+using Microsoft.Win32;
 using Microsoft.Windows.AppLifecycle;
 using Windows.ApplicationModel.Activation;
+using static Hi3Helper.Logger;
```

Comment:
This is just a small CodeQA thing, but we prefer to arrange all the usings by the alphabet. So we can move this on the bottom of line 8

Reply:
Never noticed they were ordered alphabetically in other files
changed in c0a32a64